go.mod

```diff
@@ -69,7 +69,7 @@ require (
 )
 
 require (
-	emperror.dev/errors v0.8.1 // indirect
```

Comment:
unnecessary

Reply:
Thank you for the thorough review.
I will go ahead and remove it.